app/uk/gov/hmrc/voapropertylinking/connectors/modernised/ExternalPropertyLinkApi.scala

```diff
@@ -39,6 +40,23 @@ class ExternalPropertyLinkApi @Inject()(
 )(implicit executionContext: ExecutionContext)
     extends BaseVoaConnector {
 
+  def getMyAgentPropertyLinks(
+        agentCode: Long,
+        searchParams: GetMyOrganisationPropertyLinksParameters,
+        params: Option[PaginationParams])(implicit request: RequestWithPrincipal[_]): Future[PropertyLinksWithAgents] =
```

Comment:
why is paragination parameters optional?

Reply:
removed Option for pagiantion